canonical-web-and-design/jaas-dashboard
# Add basic Nav component

## Done

Add basic Nav component

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check code
- Check basic nav displays as expected

Fixes #8
